### PR TITLE
Readability within Cascade_Layers page

### PR DESCRIPTION
**Pull request by Bchenevey**
While reading the Cascade Layers page, I came across a section that I found difficult to understand, and wanted to try to make it easier for future readers to understand the concepts.

I significantly expanded the explanation and added some additional assumptions to make in the introductory phase of this section. 

I felt that it was difficult to grasp the material in the way it was written and spent a significant amount of time re-reading the section before I was able to understand the meaning behind the message being conveyed. I expanded the section significantly but I think it will help readers understand the concepts quicker than the way it was worded previously.

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 1]
```diff
@@ -338,7 +338,15 @@ The layer order precedence is the order in which layers are created. If the auth
 9. inline important styles
 10. transitioning styles
 
-The order of layers is the order in which the layers were created which is the order in which they first appeared for normal styles, with unlayered styles coming last. For normal styles, later declared layers take precedence over earlier declared layers, with unlayered styles being in a final implicit unnamed layer. This order is inverted for important styles. Then inline styles, declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style) directly on an element, take precedence over layered and unlayered styles.
+Normal styles declared inside layers receive the lowest priority and are sorted by the order in which the layers were created. The first created layer has lowest precedence, and the layer created last has highest precedence amongst layers. Normal styles declared within `firstLayer` will be overridden by any subsequent stylings on the list if any conflicts exist.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:341]:
```suggestion
Normal styles declared inside layers receive the lowest priority and are sorted by the order in which the layers were created. Normal styles in the first created layer have the lowest precedence, and normal styles in the layer created last have the highest precedence among the layers. In other words, normal styles declared within `firstLayer` will be overridden by any subsequent stylings on the list if any conflicts exist.
```

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 2]
```diff
@@ -338,7 +338,15 @@ The layer order precedence is the order in which layers are created. If the auth
...
+
+Next up are any stylings declared outside of layers. The stylings in `C.css` were not imported to a layer and will override any conflicting styles from `firstLayer` and `secondLayer`.
+
+Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, `C.css`). Animating styles have higher precedence than inline normal styles.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:345]:
```suggestion
Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, `C.css`). 

Animating styles have higher precedence than all normal styles, including inline normal styles.
```

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 3]
```diff
@@ -338,7 +338,15 @@ The layer order precedence is the order in which layers are created. If the auth
...
+
+Important styles (`!important`) take precedence over any styles previously mentioned on our list. They are sorted inversely from normal styles. Any unlayered important styles have less precedence those declared within the layers. Important styles found within layers are, again, sorted in order of creation. For important styles, the last created layer has lowest precedence, and the first created layer has highest precedence amongst layers.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:347]:
```suggestion
Important styles (`!important`) take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared. layers.
```

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 4]
```diff
@@ -338,7 +338,15 @@ The layer order precedence is the order in which layers are created. If the auth
...
+
+Inline important styles again have higher precedence than important styles declared elsewhere, and transitioning styles have the highest precedence.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:349]:
```suggestion
Inline important styles again have higher precedence than important styles declared elsewhere. 

Transitioning styles have the highest precedence. When a normal property value is being transitioned, it takes precedence over all other property value declarations, even inline important styles; but only while transitioning.
```

**Review by estelle — COMMENTED** [review]
The edits you made are super helpful. I added some suggestions (making it even more verbose, but hopefully clearer). Let me know if you agree with the suggestions by committing them, or if you don't, by saying so ;).

**Comment by Bchenevey** [comment]
I think all of those changes are an improvement, thank you!

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 5]
```diff
@@ -338,7 +338,19 @@ The layer order precedence is the order in which layers are created. If the auth
...
+Normal styles declared inside layers receive the lowest priority and are sorted by the order in which the layers were created. Normal styles in the first created layer have the lowest precedence, and normal styles in the layer created last have the highest precedence among the layers. In other words, normal styles declared within `firstLayer` will be overridden by any subsequent stylings on the list if any conflicts exist.
+
+Next up are any styles declared outside of layers. The styles in `C.css` were not imported into a layer and will override any conflicting styles from `firstLayer` and `secondLayer`. Normal styles not declared in a layer always have higher precedence than normal-importance layered styles.
+
+Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, `C.css`).
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:345]:
```suggestion
Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, and `C.css`).
```

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 6]
```diff
@@ -338,7 +338,19 @@ The layer order precedence is the order in which layers are created. If the auth
...
+
+Next up are any styles declared outside of layers. The styles in `C.css` were not imported into a layer and will override any conflicting styles from `firstLayer` and `secondLayer`. Normal styles not declared in a layer always have higher precedence than normal-importance layered styles.
+
+Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, and `C.css`).
+
+Animating styles have higher precedence than all normal styles, including inline normal styles.
+
+Important styles (`!important`) take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared. layers.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:349]:
```suggestion
Important styles, property values that include the `!important` flag, take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared. layers.
```

**Review by estelle — APPROVED** [review]
awesome. Thank you

**Pull request merged**